ksql-rest-app/src/main/java/io/confluent/ksql/rest/util/JsonUtil.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.ksql.rest.util;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import io.confluent.ksql.GenericRow;
+
+public class JsonUtil {
+
+  private static ObjectMapper objectMapper = new ObjectMapper();
+
+  public static GenericRow buildGenericRowFromJson(final String jsonString) throws IOException {
```

Comment:
This is not nice code. We should drop this in favour of using Jackson's Bean serialization support - i.e. use StreamedRow as we were before.

Reply:
This was only used in Rest side. since I reverted changes and went back to StreamedRow I deleted this class along with the test.